### PR TITLE
Update measurements to use invoke

### PR DESCRIPTION
**Pull request by tnorling**
[msal-browser] Updates remaining measurements to use the invoke wrappers. This will also result in a few more measurements we're not collecting today.

Will follow up with msal-common changes in a separate PR

### CODE REVIEW

File: lib/msal-browser/src/controllers/StandardController.ts [hunk 1]
```diff
@@ -2010,72 +2020,75 @@ export class StandardController implements IController {
...
+                        this.logger,
+                        this.performanceClient,
                         silentRequest.correlationId
+                    )(silentRequest, requestWithCLP).catch(
+                        (refreshTokenError: AuthError) => {
+                            const isServerError =
+                                refreshTokenError instanceof ServerError;
+                            const isInteractionRequiredError =
+                                refreshTokenError instanceof
+                                InteractionRequiredAuthError;
```
Comment (peterzenz) [inline lib/msal-browser/src/controllers/StandardController.ts:2058]: nit:  Odd line break here, maybe remove the newline or indent

File: lib/msal-browser/src/controllers/StandardController.ts [hunk 2]
```diff
@@ -2010,72 +2020,75 @@ export class StandardController implements IController {
...
+                            const isInvalidGrantError =
+                                refreshTokenError.errorCode ===
+                                BrowserConstants.INVALID_GRANT_ERROR;
+
+                            if (
+                                (!isServerError ||
+                                    !isInvalidGrantError ||
+                                    isInteractionRequiredError ||
```
Comment (peterzenz) [inline lib/msal-browser/src/controllers/StandardController.ts:2066]: Can you list the or clauses in the order of most common to least in this expression?  I'm guessing interaction required would be before invalid grant but I could be mistaken.

**Review by peterzenz — APPROVED** [review]

File: change/@azure-msal-browser-4b59ec24-18be-486e-8bb9-33d5262c4a0f.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update telemetry measurements to use invoke",
```
Comment (sameerag) [inline change/@azure-msal-browser-4b59ec24-18be-486e-8bb9-33d5262c4a0f.json]:
```suggestion
  "comment": "Update telemetry measurements to use invoke #6484",
```

File: lib/msal-browser/src/controllers/StandardController.ts (hunk 1 above)
Comment (tnorling) [reply lib/msal-browser/src/controllers/StandardController.ts:2058]: This is the formatter's doing, agree it's a bit odd. We'd have to change the default rules to change the behavior here

File: lib/msal-browser/src/controllers/StandardController.ts (hunk 2 above)
Comment (tnorling) [reply lib/msal-browser/src/controllers/StandardController.ts:2066]: Do we have data on this? I'm not actually changing anything here, it's just showing in the diff due to new indentation

Comment (tnorling) [reply lib/msal-browser/src/controllers/StandardController.ts:2066]: Invalid grant will be every 24 hours for sure, not sure if interaction_required should be expected to be more or less often than that

Comment (sameerag) [reply lib/msal-browser/src/controllers/StandardController.ts:2066]: interaction_required can show for background changes like MFA or CA changes apart from the token expiry.  If we take off the policy or password changes which cannot be predicted, invalid_grant still precedes interaction_required as the later is invoked when the session expires and the former is when the RT expires (predictably 24 hrs).

File: lib/msal-browser/src/controllers/StandardController.ts (hunk 1 above)
Comment (sameerag) [reply lib/msal-browser/src/controllers/StandardController.ts:2058]: May be increasing line length limits will do it, can be a separate PR.

**Review by sameerag — APPROVED** [review]

**Pull request merged**